test/e2e/common/secrets.go

```diff
@@ -180,14 +201,19 @@ func secretForTest(namespace, name string) *api.Secret {
 	}
 }
 
-func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32) {
+func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32, secretName string) {
 	var (
 		name            = "secret-test-" + string(uuid.NewUUID())
 		volumeName      = "secret-volume"
 		volumeMountPath = "/etc/secret-volume"
 		secret          = secretForTest(f.Namespace.Name, name)
 	)
 
+	if len(secretName) != 0 {
+		// Existence of a second secret with same name and different namespace shouldn't affect the mount
```

Comment:
There is no "other secret" here. This is just a param to name the secret that it will be created. We are using it for that now, but it is just about using that name for the secret.